docs/hoon/concepts.md

```diff
@@ -53,20 +47,21 @@ The simplest noun syntax uses brackets, like `[a b]` for the cell
 of `a` and `b`.  Brackets nest right; `[a b c d]` is `[a [b [c
 d]]]`.  You'll also see braces: `{a b c d}`.
 
-A noun list, by convention, points right and is zero-terminated.
-Hoon uses tuples (improper nouns) more freely than Lisp; only
-genuinely iterative structures are null-terminated lists.
+A noun list, by convention, goes from left to right and is 
+null-terminated. That said, Hoon uses tuples (improper nouns) more 
+freely than Lisp; only genuinely iterative structures are 
+null-terminated lists.
 
 When an atom is used as a string (a `cord`), the bytes are always
-UTF-8 in LSB first order.  When you see `foo`, `$foo`, `%foo`,
+UTF-8 in LSB-first order.  When you see `foo`, `$foo`, `%foo`,
 etc, this refers to the integer `0x6f.6f66`, aka `7.303.014`.
-(Hoon syntax breaks integer atoms in the German style.)
+(Hoon syntax uses `.` to break integer atoms in the German style.)
 
 One common operation on nouns is `slot`, a tree addressing
 scheme which maps an atom to a subtree.  The whole noun is slot
 `1`; the left child of `n` is `2n`, the right child `2n+1`.
 
-Nothing in Nock, Hoon or Urbit can create cycles in a noun or
+Nothing in Nock, Hoon, or Urbit can create cycles in a noun or
```

Comment:
@joshuareagan @belisarius222 We should probably file an issue for this -- whether or not official Urbit documentation should use the Oxford comma.

Reply:
The Oxford comma is recommended by good people, people who care about precision and programmers.